Dockerfile

```diff
@@ -1,9 +1,19 @@
-FROM golang:1.11-alpine
+FROM golang:1.11-alpine as build
```

Comment:
Maybe try Go 1.13?

Reply:
Upgrading the Go version should be done in another PR as the current one should just move the Docker build to a multi stage build.